src/actions/validationActions.js

```diff
@@ -0,0 +1,25 @@
+import { updateSpinner } from './uiActions'
+import { validEthAddress } from '../helpers/validators'
+import { translate } from 'services/translations/translations'
+import { addToast } from './uiActions'
+
+export function validateAddress({ addr, dirty, validate, name }) {
+	return async function (dispatch) {
+		try {
```

Comment:
`const { authType } = getState().persist.accoun.wallet`
and pass it for `validEthAddress`
That's why I told you to make this function as action